spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :request do
+  describe 'GET #index' do
+    it 'returns a success response' do
+      get users_url
+      expect(response).to be_successful
+    end
+  end
+
+  describe 'users#show' do
+    it 'show for users works!' do
+      get '/users/1'
+      expect(response).to have_http_status(:ok)
+      expect(response.body).to include('Teacher from Mexico.')
+    end
```

Comment:
- [ ] your spec file should include tests that ensure the correct view template is rendered.
Example : `expect(response).to render_template(:show)`